Library/UIPageViewController+Extensions.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  UIPageViewController+Extensions.swift
```

Comment:
Doesn't look like we need this extension?

Reply:
Actually this extension was added after I merged the branch with the master.

Reply:
Ok, but I guess we can remove it - seems like an empty extension